moveit_simple/src/moveit_simple.cpp

```diff
@@ -478,6 +595,23 @@ Eigen::Affine3d Robot::transformToBase(const Eigen::Affine3d &in,
 
 
 
+bool Robot::getFK(const std::vector<double> & joint_point,
+                  Eigen::Affine3d &pose) const
+{
+  robot_state_->setJointGroupPositions(joint_group_, joint_point);
+  const std::vector<std::string> link_names = robot_model_ptr_->getLinkModelNames();
+  if (!link_names.empty())
```

Comment:
You should check that the number of joints is equal to the number of free joints.